src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -70,6 +70,17 @@ public static bool InStandbyMode
             }
         }
 
+        public static string InstanceId
+        {
+            get
+            {
+                string id = Environment.GetEnvironmentVariable("WEBSITE_INSTANCE_ID")
```

Comment:
There is an existing function FileTraceWriter.GetInstanceId that has some of this logic already. I think you should extract part of that out into a shared helper that you use. It also uses Environment.MachineName rather than a guid in local cases.